cdap-e2e-tests/src/e2e-test/features/controlcenter/ControlCenter.feature

```diff
@@ -0,0 +1,575 @@
+    Then Verify the deleted pipeline is not present in the control center page
+
+  @BQ_INSERT_INT_SOURCE_TEST @BQ_SINK_TEST
+  Scenario: Verify User is able to create pipeline by using preferences,entering the key and value pair and validate it deploy successfully.
+    Given Open Datafusion Project to configure pipeline
+    Then Click on the Hamburger bar on the left panel
+    Then Click on Control Center link from the hamburger menu
+    Then Click on the Plus Green Button to import the pipelines
+    Then Verify user is able to click on the create button to create a pipeline successfully
+    When Expand Plugin group in the LHS plugins list: "Source"
+    When Select plugin: "BigQuery" from the plugins list as: "Source"
+    When Expand Plugin group in the LHS plugins list: "Sink"
+    When Select plugin: "BigQuery" from the plugins list as: "Sink"
+    Then Connect plugins: "BigQuery" and "BigQuery2" to establish connection
+    Then Navigate to the properties page of plugin: "BigQuery"
+    And Enter input plugin property: "referenceName" with value: "Reference"
+    Then Click on the Macro button of Property: "project" and set the value to: "projectId"
+    And Enter input plugin property: "datasetProject" with value: "projectId"
+    And Replace input plugin property: "dataset" with value: "dataset"
+    Then Override Service account details if set in environment variables
+    And Enter input plugin property: "table" with value: "bqSourceTable"
+    Then Click on the Get Schema button
+    Then Validate "BigQuery" plugin properties
+    And Close the Plugin Properties page
+    Then Navigate to the properties page of plugin: "BigQuery2"
+    Then Click on the Macro button of Property: "project" and set the value to: "projectId"
+    Then Override Service account details if set in environment variables
+    Then Enter input plugin property: "datasetProject" with value: "projectId"
+    Then Enter input plugin property: "referenceName" with value: "BQReferenceName"
+    Then Enter input plugin property: "dataset" with value: "dataset"
+    Then Enter input plugin property: "table" with value: "bqTargetTable"
+    Then Validate "BigQuery" plugin properties
+    And Close the Plugin Properties page
+    Then Save the pipeline
+    Then Deploy the pipeline
+    Then Click on the Hamburger bar on the left panel
+    Then Click on Control Center link from the hamburger menu
+    Then Verify the user is able to set the preferences for the created pipeline in the control center page
+    Then Verify the user is able to enter the value in the key input field "keyValue"
+    Then Verify the user is able to enter the value of the key in the value input field "value"
+    Then Verify user is able to click on save and close button of set preferences
+    Then Verify user is able to click on the data pipeline added in the control center page
+    Then Run the Pipeline in Runtime
```

Comment:
If we are setting the key and value for a particular macro property field of plugin then, we need to even check by running the pipeline whether the added preferences are used while deploying.

Reply:
We are already checking it by keeping projectids in source and sink plugin as macros.